todo.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', () => {
+    let addButton = document.getElementById('addButton');
+    let ul = document.getElementById('ulTodo');
+    let inputText = document.getElementById('todoAddText')
+    let form = document.getElementById('form');
+
+    function addToList () {
+        let newListItem = document.createElement('li');
+        newListItem.innerText = inputText.value;
+        ul.appendChild(newListItem);
+        form.reset(); 
+    }
+
+    function emptyError () {
+        let errorP = document.createElement('p')
+        errorP.innerText = 'Error, please input text!'
+        form.appendChild(errorP)
+    }
+
```

Comment:
Currently your code is adding a new error message to the form every time there is an error. You could fix this by creating your `p` element and appending it to the form on line 6. 

Your `emptyError` function could then utilize if/else statements to control the innerText of your `errorP` variable.

Let me know if this makes sense or if you have any questions about this!